error id: ZjWoNYCCx5sm32jEAnv9zQ==
### Bloop error:

called from private utest.runner.BaseRunner.makeTask(sbt.testing.TaskDef)sbt.testing.Task
#### Short summary: 

called from private utest.runner.BaseRunner.makeTask(sbt.testing.TaskDef)sbt.testing.Task